src/components/themed/ThemedTicker.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from 'react'
+import { Platform } from 'react-native'
+import Ticker from 'react-native-ticker'
+
+import { useTheme } from '../services/ThemeContext.js'
+
+type Props = {
+  children: React.Node,
+  style: any,
+  numberOfLines?: number,
+  ellipsizeMode?: string,
+  durations?: number,
+  [x: string]: any
+}
+
+export const ThemedTicker = ({ children, style, duration, ...props }: Props) => {
+  const theme = useTheme()
+
+  // TODO: The nature of the Ticker compoenent does not apply textProps to the string as a whole so these propds dont work.
+  // May not be able to have the text automatically adjust to fit with this ticker. If this is needed, may need more creative solution.
+  const iosTextProps = { numberOfLines: props.numberOfLines || 1, adjustsFontSizeToFit: true }
+  const androidTextProps = { numberOfLines: props.numberOfLines || 1, ellipsizeMode: props.ellipsizeMode || 'middle' }
+  const textProps = Platform.OS === 'ios' ? iosTextProps : androidTextProps
+
+  const defaultTickerStyle = {
+    color: theme.primaryText,
+    fontFamily: theme.fontFaceDefault,
+    fontSize: theme.rem(1)
+  }
+
+  return (
+    <Ticker {...props} textProps={textProps} textStyle={[defaultTickerStyle, style]} duration={duration || 500}>
+      {children}
+    </Ticker>
```

Comment:
Two issues with the themed ticker: 

1. The ticker dependency itself had to be modified in node_modules in order to work, 
which means we will need to fork the repo for this module and apply a custom update to work around a bug with it. 

2. There is a noticable lag time for the numbers to render in my simulator. User has to wait about a second for the numbers inside this component to render. This lag time might be much quicker on real devices. Test this further. Might need to update the forked library to improve performance of the initial render.

Reply:
the simulator is pretty slow when using a debug build. try a release build